pkg/webreg/webreg.go

```diff
@@ -236,6 +238,79 @@ const templateDefinitions = `
 	<nobr><a href="/workflow/{{ . }}" style="font-family:monospace">{{ . }}</a></nobr>
 {{ end }}
 
+{{ define "referenceProperties" }}
+  <table class="table">
+  <thead>
+  <tr>
+    <th title="Property name" class="info">Property</th>
+    <th title="Property name" class="info">Value</th>
+    <th title="Property name" class="info">Description</th>
+  </tr>
+  </thead>
+  <tbody>
+  {{ if .ActiveDeadlineSeconds }}
+    <tr>
+      <td>Step timeout</td>
+      <td>{{ .ActiveDeadlineSeconds }} seconds</td>
+      <td>Limits the execution time of the step.</td>
+    </tr>
+  {{ end }}
+  {{ if .TerminationGracePeriodSeconds }}
+    <tr>
+      <td>Termination grace period</td>
+      <td>{{ .TerminationGracePeriodSeconds }} seconds</td>
+      <td>Period of time until SIGKILL signal is sent to the test pod (after SIGTERM signal is sent).</td>
```

Comment:
Can we TODO to link to the timeouts page when it's ready?

Reply:
Sure!

Reply:
https://github.com/openshift/ci-tools/pull/1599